buildscripts/checkstyle.xml

```diff
@@ -89,28 +89,6 @@
       <property name="tokens"
         value="LITERAL_DO, LITERAL_ELSE, LITERAL_FOR, LITERAL_IF, LITERAL_WHILE"/>
     </module>
-    <module name="LeftCurly">
```

Comment:
This checkstyle rule conflicts with spotless formatting of case blocks. I just removed it, and perhaps other formatting-related ones can be removed as well later since they're at best redundant with spotless and at worse may conflict.